### PR TITLE
docs: Added seed data information

### PR DESCRIPTION
**Pull request by elitan**
https://docs-git-docs-seed-nhost.vercel.app/platform/database#seed-data

### CODE REVIEW

File: docs/docs/platform/database/index.mdx [hunk 1]
```diff
@@ -131,6 +131,21 @@ To track database changes, use the [Nhost CLI](/platform/cli) to develop locally
 
 Learn how to [get started with Nhost CLI](/platform/overview/get-started-with-nhost-cli).
 
+## Seed Data
+
+Seed data is a way of automatically adding data to your database when you create a new environment. This is, for the moment, only applicable when you're using the [Nhost CLI](/platform/cli) to develop locally. When you're running `nhost up` for the first time, seed data is added.
+
+In the future, seed data will also be added to new preview environments.
+
+Seed data should be located in `nhost/seeds/default/` and are executed in alphabetical order.
```
Comment (plmercereau) [inline docs/docs/platform/database/index.mdx:140]: in alphanumeric order? When reading "alphabetical" I think "a,b,c" whereas my brain is telling me to write down seed file names starting with "01, 02, 03"

**Review by plmercereau — APPROVED** [review]

File: docs/docs/platform/database/index.mdx (hunk 1 above)
Comment (elitan) [reply docs/docs/platform/database/index.mdx:140]: True. Do we have a better word we can use? Or maybe skip the order completely as seed scripts should maybe be able to run in any order?

Comment (plmercereau) [reply docs/docs/platform/database/index.mdx:140]: I can't think of something else than "alphanumeric"
depending on how seeds are written (and how the CLI loads them - single transaction, or one transaction per file), yes, order matter. For instance, if one file has some seeds for a `todo` table with a `user_id` column, the other file with the `users` should be seeded first

Comment (elitan) [reply docs/docs/platform/database/index.mdx:140]: I added an example showing numbers can go first in alphabetic order.

**Pull request merged**